torch/csrc/jit/mobile/import.cpp

```diff
@@ -146,7 +145,7 @@ c10::intrusive_ptr<c10::ivalue::Object> objLoaderMobile(
     custom_class_type->getMethod("__setstate__").run(stack);
     return obj;
   } else {
-    auto dict = std::move(input).toGenericDict();
+    auto dict = input.toGenericDict();
```

Comment:
Likewise what was wrong before here?

Reply:
input is const IValue&, moving it has no effect.